lecsum-fe/services/api_client.py

```
@@ -97,3 +97,27 @@ def grade_quiz(self, quiz_set_id: int, quiz_id_list: list[int], user_answer_list
         res = requests.post(url, json=payload, timeout=self.timeout)
         res.raise_for_status()
         return res.json()
+    
+    def get_quiz_sets(self, document_id: int) -> Dict[str, Any]:
+        """퀴즈 목록 불러오기"""
+        params = {"document_id": document_id}
+        response = requests.get(f"{self.base_url}/api/quizzes/quiz-sets", params=params)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)